### ISSUE TITLE
Functional init.d file

### ISSUE THREAD

**Issue by EZarnosky** [theotherp/nzbhydra#119]
Any chance of creating a init.d file to include? I have spent a lot of time trying to get start-stop-daemon to work but had no luck, I even tried the sample on http://www.htpcguides.com/install-nzbhydra-ubuntu-universal-usenet-searching/

Would also be nice to have switches for a log file and controlling where the pidfile is located.

**Comment by theotherp** [comment]
Yes to the switches, no to the init.d file. At least not by me. I know enough linux to keep my server running (barely) but that's it.

Perhaps you could describe what problems you had.

**Comment by theotherp** [comment]
By the way, you can set the (full) path for the log file in the config.

**Comment by theotherp** [comment]
Uhm, I just realized something. I use custom exit codes to signal the underlying process if it should be restarted, shutdown etc. That might cause problems, will have to look into that.

**Comment by theotherp** [comment]
I just tried the init.d script from HTPC guides and it worked flawlessly for me.

**Comment by EZarnosky** [comment]
Well if you get the switches in place, I'll see if I can get a init.d and defaults file whipped up.

I am not using the default directories that they have in the HTPC Guides page. I've modified the paths and for some reason it doesn't work. After playing with it a couple hours I said enough was enough. It would be easier and cleaner if I was able to create a init.d with supported switches.

Regardless, you have nice app. It is a great replacement for NZBMegaSearch.
